app/scrapers/sachsen_scraper.rb

```diff
@@ -0,0 +1,187 @@
+module SachsenScraper
+  BASE_URL = 'http://edas.landtag.sachsen.de'
+  SEARCH_FIELD = 'ctl00$masterContentCallback$content$suchmaske$tblSearch$tabSuche$panelUmSuchmaskeEinfach$suchmaskeEinfachCallback'
+  SEARCH_FIELD2 =
+    'ctl00_masterContentCallback_content_suchmaske_tblSearch_tabSuche_panelUmSuchmaskeEinfach_suchmaskeEinfachCallback_sb_Einf'
+
+  def self.extract_overview_items(page)
+    page.search('//td[@class="dxdvItem_EDAS"]/table')
+  end
+
+  def self.extract_title(item)
+    item.search('.//tr[3]/td/b').text.strip
+  end
+
+  def self.extract_meta_text(item)
+    text = item.search('.//tr[3]/td').text.split("\n").find{ |t| t.include?('KlAnfr') || t.include?('GrAnfr') }
+    text.strip if !text.nil?
+  end
+
+  def self.extractType(text)
+    if (text[0..5] == 'KlAnfr')
+      Paper::DOCTYPE_MINOR_INTERPELLATION
+    else
+      Paper::DOCTYPE_MAJOR_INTERPELLATION
+    end
+  end
+
+  def self.extract_date(meta_text)
+    Date.parse(meta_text.match(/\d+\.\d+\.\d{4}/).to_s)
+  end
+
+  def self.extract_reference(meta_text)
+    meta_text.split('Drs')[1].strip
+  end
+
+  def self.extract_originators(meta_text)
+    people = [meta_text.partition(/\p{Upper}{2}/)[0].strip]
+    parties = [meta_text.match(/\p{Upper}\p{Lower}*\p{Upper}+\s*\p{Upper}*/)[0].strip]
+    originators = { people: people, parties: parties}
+  end
+
+  def self.extract_overview_paper(item, doctype)
+    meta_text = extract_meta_text item
+    meta_text = meta_text[6..meta_text.length].strip
+    full_reference = extract_reference(meta_text)
+    {
+      legislative_term: full_reference.split('/')[0],
+      full_reference: full_reference,
+      reference: full_reference.split('/')[1],
+      doctype: doctype,
+      title: extract_title(item),
+      url: nil,
+      published_at: extract_date(meta_text),
+      originators: extract_originators(meta_text),
+      is_answer: nil,
+    }
+  end
+
+  def self.extract_detail_paper(item)
+    meta_text = extract_meta_text(item)
+    type = extractType(meta_text)
+    extract_overview_paper(item, type)
+  end
+
+  def self.switchToTerm(i, m, top)
+    nav = top.frame_with(name: 'head').click
+    nav = m.get("/redirect.aspx?wahlperiode=" + i.to_s)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.